docs/book/helpers/html-attributes.md

```diff
@@ -0,0 +1,17 @@
+# HtmlAttributes
+
+The `HtmlAttributes` helper is used to create `HtmlAttributesSet` objects.
+
+## Basic Usage
+
+```php
+$exampleAttributes = $this->htmlAttributes();
+$exampleAttributes->add('class', 'example');
+<div<?= $exampleAttributes ?>>
+```
+
+Output:
+
+```html
+<div class="example">
+```
```

Comment:
The description of the class `Laminas\View\HtmlAttributesSet` is missing. All methods of the class (`set`, `add`, `merge`, `hasValue`, `__toString`) must be described and each with a code example. Also the fact that this class extends PHP's `ArrayObject` – with link to the related PHP documentation.

Reply:
You can adapt the structure from the view helper pages of laminas-i18n. For example:

* https://docs.laminas.dev/laminas-i18n/view-helpers/currency-format/
* https://docs.laminas.dev/laminas-i18n/view-helpers/date-format/
* https://docs.laminas.dev/laminas-i18n/view-helpers/number-format/